tests/core/conversations_worker/integration/test_conversation_integration.py

```
@@ -0,0 +1,467 @@
+"""
+Integration tests for the M2 Conversation Worker.
+
+Requires a running Holmes server with ENABLE_CONVERSATION_WORKER=true and:
+    ROBUSTA_UI_TOKEN   – base64 JSON with Supabase credentials
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)